interactions/api/http/interaction.py

```diff
@@ -1,6 +1,10 @@
 from typing import TYPE_CHECKING, List, Optional, Union
 
+from aiohttp import MultipartWriter
+
+from ...utils.missing import MISSING
```

Comment:
We don't use `MISSING` in the http files

Reply:
Applied.